client-react/src/App.js

```diff
@@ -1,23 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import ProductLine from './ProductLine';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ProductLine text={'banane'}/>
```

Comment:
`{}` c'est pour passer une expression javascript en prop, comme un nom de variable `{productName}` ou le retour d'une fonction `{getProductName()}`.
Ici on a juste besoin de passer la chaine de caractères `"banane"` donc pas besoin des accolades. Ça marche parce qu'en javascript, une chaine de caractère est une expression, mais on peut écrire plus simplement :

```suggestion
      <ProductLine text="banane" />
```

Reply:
la correction a été apporté. Je m'aperçois que c'est webstorm qui mets tout seul ces `{}` ^^